docs/usage.md

```diff
@@ -69,119 +115,360 @@ cat ./compilation.json
+### Cross-Compilation<a name="cross-compilation"></a>
 Cross-compilers are auto-detected by CodeChecker, so the `--target` and the
-compiler pre-configured
-include paths of `gcc/g++` are automatically passed to `clang` when analyzing.
+compiler pre-configured include paths of `gcc/g++` are automatically passed to
+`clang` when analyzing.
 
 **Make sure that the compilers used for building the project (e.g.
 `/usr/bin/gcc`) are accessible when `CodeChecker analyze` or `check` is
 invoked.**
 
-### Incremental Analysis <a name="incremental-analysis"></a>
+## Step 3: View analysis results in command line or generate static HTML files<a name="step-3"></a>
+You can print detailed results (including the control flow) in command line by
+running:
+
+```sh
+CodeChecker parse --print-steps ./reports
+...
+Found no defects in divide.c
+[STYLE] .../docs/examples/src/main.c:15:7: do not use 'else' after 'return' [readability-else-after-return]
+    } else {
+      ^
+  Report hash: c120e732461d8e478c35d1940ef53488
+  Steps:
+    1, main.c:15:7: do not use 'else' after 'return'
+
+[STYLE] .../docs/examples/src/main.c:16:43: 10 is a magic number; consider replacing it with a named constant [readability-magic-numbers]
+        test_case = strtol(argv[1], NULL, 10);
+                                          ^
+  Report hash: 17e5ce988ab2cfd6284d3ec7c66904a0
+  Steps:
+    1, main.c:16:43: 10 is a magic number; consider replacing it with a named constant
+
+Found 2 defect(s) in main.c
+...
+```
+
+It is possible to generate reports as plain `HTML` files using the
+`CodeChecker parse` command. (For other output formats please consult with help
+of `parse` subcommand.)
+
+```sh
+CodeChecker parse --export html --output ./reports_html ./reports
+...
+To view the results in a browser run:
+> firefox ./reports_html/index.html &
+```
+
+`./reports_html` directory will contain an `index.html` with a link to all
+findings that are stored in separate `HTML` files (one per analyzed build
+action).
+
+![Analysis results in static HTML files](images/static_html.png)
+
+## Step 4: Incremental Analysis<a name="step-4"></a>
 The analysis can be run for only the changed files and the `report-directory`
 will be correctly updated with the new results.
 
 There are two supported ways for incremental analysis.
 
-a) In case you can build your
-project incrementally, you can build, log and analyze only the changed files and all the files
-that are depending on the source code changes (in case of the update of a header file).
-
-b) If you only want to re-analyze changed source files, without re-building your
-project, you can use skip list to tell CodeChecker which files to analyze.
+a) In case you can build your project incrementally, you can build, log and
+analyze only the changed files and all the files that are depending on the
+source code changes (in case of the update of a header file).
 
+b) If you only want to re-analyze changed source files, without re-building
```

Comment:
A link would be good here what is a skip file, how it looks like: https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/user_guide.md#skip

Reply:
Done.